### PR TITLE
Feature/search service

### PR DESCRIPTION
**Pull request by aagavin**
Added search page and search service
this closes issue #4

### CODE REVIEW

File: src/providers/search.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      else {
+        resolve([]);
+      }
+    })
+  }
+
+  private _parseResponse(imdbResponse: string, query: string): Array<Object> {
+    let results = Array<Object>();
+    // replace spaces the query string with _
+    query = (query).split(' ').join('_')
```
Comment (LeontiBrechko) [inline src/providers/search.ts]: @aagavin, please, put a semicolon on each statement line in order to eliminate possible bugs. The same applies to lines 34, 36, 43 of this file

**Comment by LeontiBrechko** [comment]
Search doesn't work because of CORS (cross-origin resource sharing) neither in chrome nor on ios device.

**Review by LeontiBrechko — APPROVED** [review]

**Comment by eddieysong** [comment]
Seems good to me.

**Review by eddieysong — APPROVED** [review]
Everything seems good, thank you!

**Pull request merged**